components/service/glean/src/test/java/mozilla/components/service/glean/storages/TimingDistributionsStorageEngineTest.kt

```diff
@@ -128,6 +135,39 @@ class TimingDistributionsStorageEngineTest {
             snapshot["telemetry.test_timing_distribution"]?.toJsonObject().toString())
     }
 
+    @Test
+    fun `serializer should serialize timing distribution that matches schema`() {
+        val ping1 = PingType("store1", true)
+
+        val server = getMockWebServer()
```

Comment:
Instead of setting up the whole ping machinery here and generating a ping, what if we had a test in GleanTest.kt that adds every single metric type and checks the schema ? (can be done as a follow-up). Alternatively, what about we call in the `StorageEngineManager` and get the resulting payload here?